resources/definitions/dagoma_neva.def.json

```diff
@@ -43,14 +43,20 @@
             "default_value": "elliptic"
         },
         "machine_gcode_flavor": {
-            "default_value": "RepRap (RepRap)"
+            "default_value": "RepRap"
         },
         "machine_start_gcode": {
-            "default_value": ";Gcode by Cura\nG90\nG28\nM109 S100\nG29\nM104 S{material_print_temperature_layer_0}\nG0 X0 Y-85\nG0 Z0.26\nM109 S{material_print_temperature_layer_0}\nM82\nG92 E0\nG1 F200 E6\nG92 E0\nG1 F200 E-3.5\nG0 Z0.15\nG0 X10\nG0 Z3\nG1 F6000\n"
+            "default_value": ";Gcode by Cura\nG90\nG28\nM107\nM109 R100\nG29\nM109 S{material_print_temperature_layer_0} U-55 X55 V-85 Y-85 W0.26 Z0.26\nM82\nG92 E0\nG1 F200 E6\nG92 E0\nG1 F200 E-3.5\nG0 Z0.15\nG0 X10\nG0 Z3\nG1 F6000\n"
         },
         "machine_end_gcode": {
             "default_value": "\nM104 S0\nM106 S255\nM140 S0\nG91\nG1 E-1 F300\nG1 Z+3 E-2 F9000\nG90\nG28\n"
         },
+        "material_diameter": {
```

Comment:
`material_diameter` is now in the extruder definitions.

Reply:
Even if it's not overriding anything? I was under the impression that on a single extruder machine, this could be set as a machine property.

Reply:
This is true on any machine. If a value is not found in the current extruder-stack, Cura will continue looking for a value in the global stack. Right? Or was that changed?

So if you want to set the value for all extruders, setting it globally is the smartest approach.

Reply:
Having it globally caused issues in resolving the correct values in the preferences window.

Reply:
Fixed in ed1e47d